modules/acquia_cms_site_studio/config/pack_acquia_cms_core/cohesion_elements.cohesion_component.cpt_modal_window.yml

```diff
@@ -1554,7 +1559,7 @@ json_values: |
       }
   }
 json_mapper: '{}'
-last_entity_update: entityupdate_0036
+last_entity_update: entityupdate_0041
```

Comment:
@rajeshreeputra Please revert this back.

Reply:
updated